MonoGame.Framework/SDL/SDL2.cs

```diff
@@ -740,13 +797,14 @@ public static string GetName(IntPtr gamecontroller)
 
     public static class Haptic
     {
-        public const uint Infinity = uint.MaxValue;
+        public const uint Infinity = 4292967295U;
```

Comment:
That was just a test to make sure every ranges were fine, but I admit it's kind of useless.